mypy-run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -eux
 
-EXCLUDE='(^example/|^ez_setup\.py$)'
+EXCLUDE='(^example/|^ez_setup\.py$|^setup\.py$)'
```

Comment:
`setup.py` imports `ez_setup.py`, so we're explicitly excluding it now. If we want Mypy checking on `setup.py`, we'll have to figure out a way to re-include it and exclude consideration of `ez_setup.py`, because that's _never_ going to pass validation.